src/main/default/lwc/sectionTitle/__test__/sectionTitle.test.js

```diff
@@ -0,0 +1,39 @@
+import { createElement } from 'lwc';
+import SectionTitle from 'c/sectionTitle';
+
+function createTestComponent() {
+  const element = createElement('c-section-title', {
+    is: SectionTitle
+  });
+  document.body.appendChild(element);
+
+  return element;
+}
+
+// Helper function to wait until the microtask queue is empty.
+function flushPromises() {
+  // eslint-disable-next-line no-undef
+  return new Promise((resolve) => setImmediate(resolve));
+}
+
+describe('c-section-title', () => {
+  afterEach(() => {
+    // The jsdom instance is shared across test cases in a single file so reset the DOM
+    while (document.body.firstChild) {
+      document.body.removeChild(document.body.firstChild);
+    }
+  });
+
+  it('should display the title text', async () => {
```

Comment:
If having a placeholder label is part of the component functionality, a test should be added to ensure it displays properly.

Reply:
@lbergeron I have a quick question here, when performing tests on the labels we do comparaison with the name  of the label right ? 
so for example we will have smthg like this : 
`const expectedLabel = 'Random_Label_Name';`
And then we do the  comparaison like that
`expect(titleNode.textContent).toBe(expectedLabel);`

Just making sure how we perform tests on the labels, do we compare the value of the label or the name of the label like the example above?

Reply:
I might be misunderstanding your question, but I'd say that `titleNode.textContent` should match `element.title` in all cases. Since the `title` property is initialized with the label, and not the label name, the property should return the localized value.

Did you try and output the value of `element.title` right after calling `createTestComponent()` to validate whether it really gets the label as it would when running in Salesforce?

Reply:
Exactly I tried the output of the value `element.title` right after calling `createTestComponent()` and the thing is it returned the name of the label  and not the the value of the label like it happens inside salesforce, so thats why I asked this question

Reply:
Thanks For the feedback, I added the test for the placeholder label👌

Reply:
I guess the test framework does not load the localized strings and fallback on the label name. Most localization frameworks use this logic when a localized string is not found.